arrow-arith/src/aggregate.rs

```diff
@@ -410,14 +413,49 @@ where
     }
 }
 
+/// Helper to compute min/max of [`GenericByteViewArray<T>`].
+/// The specialized min/max leverages the inlined values to compare the byte views.
+fn min_max_view_helper<T: ByteViewType>(
+    array: &GenericByteViewArray<T>,
+    swap_cond: cmp::Ordering,
+) -> Option<&T::Native> {
+    let null_count = array.null_count();
+    if null_count == array.len() {
+        None
+    } else if null_count == 0 {
+        let min_idx = (0..array.len()).reduce(|acc, item| {
+            let cmp = unsafe { compare_byte_view_unchecked(array, acc, array, item) };
```

Comment:
I wonder if using `idx` rather than `item` would make it clearer that `item` is an array index and would also be more consistent with `min_idx` naming

Also, I think we should justify the `unsafe` in a comment as good practice. Something like

```suggestion
            // SAFETY:  array's length is correct so item is within bounds
            let cmp = unsafe { compare_byte_view_unchecked(array, acc, array, item) };
```